### ISSUE TITLE
Feature: Unable to pass in tsconfig path (potentially not reading `typeCoverage` in package.json)

### ISSUE THREAD

**Issue by tomardern** [alexcanessa/typescript-coverage-report#56]
Firstly, great project! I'm looking forward to using this within the organisation in which I work, and potentially contributing to the project too!

Sadly, I am having a little difficultly getting this working, hopefully you can help.

**Describe the bug**
Our project structure is as follows:
```
package.json
src/
  - app-component/app.ts ... etc
  - tsconfig.app.json
```

If I run `type-coverage` eg `type-coverage --project src/tsconfig.app.json`, then I get the following result:
```
77636 / 88900 87.32%
type-coverage success.
```

If I add the following to `package.json`, then I also get the result:
```
  "typeCoverage": {
    "project": "src/tsconfig.app.json"
  }
```
result:
```
77636 / 88900 87.32%
type-coverage success.
```

However, when I run `typescript-coverage-report`, (assuming it'll take the config from package.json as per https://github.com/alexcanessa/typescript-coverage-report#usage), then I get the following:

```
> typescript-coverage-report

┌───────────────┬───────────────────┬───────────┬─────────────┬───────────────┐
│ filenames (0) │ percent (100.00%) │ total (0) │ covered (0) │ uncovered (0) │
│ ---           │               --- │       --- │         --- │           --- │
└───────────────┴───────────────────┴───────────┴─────────────┴───────────────┘
View generated HTML Report at {project}/coverage-ts/index.html
```

**To Reproduce**
See above for detailed info, happy to refine if needed. 

**Expected behaviour**
Running `typescript-coverage-report` should return back the same type coverage as the `type-coverage` repo.

**Screenshots**
N/a

**Environment**
 - Tool version: 0.5.1 (type-coverage @2.17.2)
 - OS: Mac
 - Node version: v14.5.0
 - Browser (if applicable): [e.g. chrome, safari]

**Additional context**
N/a

**Comment by tomardern** [comment]
I've investigated this myself, and changing "." to be the path to our tsconfig file on this line: https://github.com/alexcanessa/typescript-coverage-report/blob/d188b7e8654dc0e0eafe75d36b63c88d4865c40a/src/lib/getCoverage.ts#L39

Seems to fix the issue.

**Comment by alexcanessa** [comment]
Hello @tomardern well spotted! I'm pleased you're enjoying the project!

Are you willing to fix this yourself? Otherwise, I could have a look later today.

**Comment by tomardern** [comment]
@alexcanessa If you wouldn't mind fixing yourself, that would be great, thank you for quick response.

Not sure I'm going to have time over the next few days to better understand the project, create the PR etc :(

**Comment by alexcanessa** [comment]
@all-contributors add @tomardern for bugs

**Comment by tomardern** [comment]
I've created a Pull Request for this here - https://github.com/alexcanessa/typescript-coverage-report/pull/58

**Comment by alexcanessa** [comment]
AMAZING, Thanks so much @tomardern

**Comment by alexcanessa** [comment]
Closed by #58
